app/reducers/snackbar.js

```diff
@@ -156,7 +156,7 @@ const messages = defineMessages({
   },
   CHANGEPASSPHRASE_FAILED: {
     id: "settings.errors.changePassphraseFailed",
-    defaultMessage: "{originalError}"
+    defaultMessage: "Entered incorrect private passphrase reset blocked."
```

Comment:
This message doesn't make any sense

Reply:
When you reset the password you are asked for three options first being your old private password and if that is wrong this message comes up.

Maybe its better if it is "Password reset failed entered incorrect previous passphrase"?

Reply:
I think you're being too verbose.  

"Update password failed.  Incorrect private passphrase, please try again."